packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -427,6 +427,10 @@ export class Dialog
     deactivateFocusTrap(this);
   }
 
+  onFocusTrapDeactivate(): void {
```

Comment:
I noticed components close on blur or when clicking outside or on another element (e.g., `scrim`). Can you confirm all component-closing `onFocusTrapDeactivate` calls are needed?

Reply:
I think in dialog/modal we also have to consider the value of `outsideCloseDisabled`.